src/components/Button.tsx

```diff
@@ -30,6 +30,7 @@ interface ButtonProps extends HTMLMotionProps<'button'> {
   transparent?: boolean
   squared?: boolean
   submit?: boolean
+  narrow?: boolean
```

Comment:
It's reducing the height no? Let's maybe use `short` instead?

Reply:
Yes I saw that... then `short` is really not that bad no? 
As you wish, it was just a suggestion, I let you decide.